client/src/components/Navbar/Navbar.tsx

```diff
@@ -133,7 +133,9 @@ const Navbar: React.FC = () => {
       container
     >
       <Grid xs={4} md={6} item>
-        <img className={classes.navbarLogo} src={lovingSitterLogo} />
+        <NavLink className={classes.navbarItem} to={'/'}>
+          <img className={classes.navbarLogo} src={lovingSitterLogo} />
```

Comment:
Let's also add an 'alt' attribute to HTML image elements for accessibility reasons